apps/client/src/features/viewers/minimal-timer/MinimalTimer.tsx

```diff
@@ -133,14 +133,18 @@ export default function MinimalTimer(props: MinimalTimerProps) {
   const isNegative = (time.current ?? 0) < 0;
   const showFinished = isNegative && !userOptions?.hideOvertime;
   const showEndMessage = time.current < 0 && general.endMessage && !hideEndMessage;
+  const countUp =
+    time.duration - time.current > time.duration ? time.duration * -1 + time.current : time.duration - time.current;
```

Comment:
Not really sure about the logic here, should count up not simply be the time since the timer started?

Reply:
i changed it up to use time.elapsed (stupid me hasn't seen that this exist) apart from that this logic negates the time when it goes over the duration. I do that to trigger the overtime class from the timer component wich doesn't show the negative sign before the time.

Reply:
Actually don't really know what the behaviour should be once we go overtime. Maybe keep counting but show in red?